tests/unit/options.js

```diff
@@ -736,11 +736,18 @@ exports.undef = function (test) {
 
   // block scope cannot use themselves in the declaration
   TestRun(test)
-    .addError(1, "'a' was used before it was declared, which is illegal for 'let' variables.")
-    .addError(2, "'b' was used before it was declared, which is illegal for 'const' variables.")
+//    .addError(1, "'a' was used before it was declared, which is illegal for 'let' variables.")
```

Comment:
I'd like to add an explicit `TODO` here to more clearly describe the intention of the commented-out assertions. Something like, "JSHint does not currently enforce the correct temporal dead zone semantics in this case." Sound good?